tools/astarte_import/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM elixir:1.8.2-slim as builder
+
+RUN apt-get -qq update
+RUN apt-get -qq install git build-essential curl
+
+RUN mix local.hex --force && \
+    mix local.rebar --force && \
+    mix hex.info
+
+WORKDIR /app
+ENV MIX_ENV prod
+ADD . .
+RUN mix deps.get
+RUN mix release --env=$MIX_ENV
+
+# Note: it is important to keep Debian versions in sync, or incompatibilities between libcrypto will happen
+FROM debian:stretch-slim
+RUN apt-get -qq update
+
+# Set the locale
+ENV LANG C.UTF-8
+
+# We need SSL
+RUN apt-get -qq install libssl1.1
+
+WORKDIR /app
+COPY --from=builder /app/_build/prod/rel/astarte_import .
+
+#CMD ["./bin/astarte_import", "eval", "Astarte.Import.CLI.main()", ${realm}, ${file}]
+
+ENV REALM "default"
+ENV XML_FILE "/files/data.xml"
+
+CMD ["sh", "-c", "./bin/astarte_import eval \"Astarte.Import.CLI.main()\" ${REALM} ${XML_FILE}"]
```

Comment:
I would consider an ENTRYPOINT with a bash script here, especially in case we need more complicated logic in the future and/or pass custom stuff to the command.